backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/slider/BpkSlider.kt

```diff
@@ -0,0 +1,81 @@
+/*
+ * Backpack for Android - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package net.skyscanner.backpack.compose.slider
+
+import androidx.compose.foundation.interaction.MutableInteractionSource
+import androidx.compose.material.ExperimentalMaterialApi
+import androidx.compose.material.RangeSlider
+import androidx.compose.material.Slider
+import androidx.compose.material.SliderDefaults
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import net.skyscanner.backpack.compose.theme.BpkTheme
+
+@Composable
+fun BpkSlider(
+  value: Float,
+  onValueChange: (Float) -> Unit,
+  modifier: Modifier = Modifier,
+  enabled: Boolean = true,
+  valueRange: ClosedFloatingPointRange<Float> = 0f..1f,
```

Comment:
To align with other platforms rather than having a range here can you have two properties:
`minValue: Float = 0f`, `maxValue: Float = 1f` and then map that internally to the `valueRange`

We also need to expose the `steps` property. It'd be nice to then have a sample for that as well

Reply:
I think @bvitaliyg advised not to include the steps property, as there isn't an example of that in Figma, unless I misunderstood. I'm happy to add the property and a sample though if needed!